onadata/apps/logger/signals.py

```diff
@@ -21,3 +56,41 @@ def pre_delete_attachment(instance, **kwargs):
         attachment.media_file.delete()
     except Exception as e:
         logging.error('Failed to delete attachment: ' + str(e), exc_info=True)
+
+
+@receiver(post_save, sender=Attachment)
+def update_user_profile_attachment_storage_bytes(instance, created, **kwargs):
+    """
+    Update the attachment_storage_bytes field in the UserProfile model
+    when an attachment is added
+    """
+    if not created:
+        return
+    # Probably not necessary right now, but in case needed in the future
+    if getattr(instance, 'defer_counting', False):
+        return
```

Comment:
we probably will have to implement this before deploying to production, to avoid a situation like https://github.com/kobotoolbox/kobocat/issues/490. i'll open a separate issue to avoid delaying this PR.

Reply:
apparently not! we could handle it in this PR